universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java

```diff
@@ -223,7 +223,8 @@ private ContainerTag renderQuestionSummary(
             Styles.P_2,
             Styles.PT_4,
             Styles.BORDER_B,
-            Styles.BORDER_GRAY_300);
+            Styles.BORDER_GRAY_300)
+        .attr("style", "word-break:break-word");
```

Comment:
Looks like an important improvement!

Please use the tailwind style system whenever possible. In this case I believe you can add this constant to the class list to get the same result https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/views/style/Styles.java#L2526

Reply:
tailwind docs: https://tailwindcss.com/docs/word-break

Reply:
We tried this tailwind property already, noted it in the PR description above:

Tailwind's break-words property does not actually behave as expected and does not solve the issue (looks like this property is not maintained by tailwind anymore). It does not properly wrap the long word text -- The URL link still overflows off the container. Seems like others experienced this problem: https://github.com/tailwindlabs/tailwindcss/issues/835

Break-words from tailwind adds this css:
{overflow-wrap: break-word;}

The custom css we use adds
{word-break: break-word}

Seems like older versions of tailwind worked properly for this use case, but the newer versions are causing this issue for folks.

Reply:
Ah! So sorry for not reading the description properly.

Reply:
No worries!